app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -261,9 +261,12 @@ class BookmarkFragment : LibraryPageFragment<BookmarkNode>(), UserInteractionHan
     }
 
     private suspend fun loadBookmarkNode(guid: String): BookmarkNode? = withContext(IO) {
-        requireContext().bookmarkStorage
-            .getTree(guid, false)
-            ?.let { desktopFolders.withOptionalDesktopFolders(it) }
+        // Only runs if the fragment is attached same as [runIfFragmentIsAttached]
```

Comment:
How come we couldn't just call runIfFragmentIsAttached?

Reply:
it's because the return type `BookmarkNode?`